exporter/collector/logs.go

```diff
@@ -43,20 +62,149 @@ func (l *LogsExporter) Shutdown(ctx context.Context) error {
 }
 
 func (l *LogsExporter) PushLogs(ctx context.Context, ld pdata.Logs) error {
+	mapper := &metricMapper{} // Refactor metricMapper to map MRs for logging?
+	logPushErrors := []error{}
 	for i := 0; i < ld.ResourceLogs().Len(); i++ {
 		rl := ld.ResourceLogs().At(i)
-		mapper := &metricMapper{} // Refactor metricMapper to map MRs for logging?
 		mr, _ := mapper.resourceToMonitoredResource(rl.Resource())
 		for j := 0; j < rl.InstrumentationLibraryLogs().Len(); j++ {
 			ill := rl.InstrumentationLibraryLogs().At(j)
 			for k := 0; k < ill.LogRecords().Len(); k++ {
 				log := ill.LogRecords().At(k)
-				l.logger.Log(logging.Entry{
-					Resource: mr,
-					Payload:  log.Body().AsString(),
-				})
+				entry, err := l.logToEntry(log, make(map[string]interface{}), mr)
+				if err != nil {
+					logPushErrors = append(logPushErrors, err)
+				} else {
+					l.logger.Log(entry)
+				}
 			}
 		}
 	}
+	if len(logPushErrors) > 0 {
+		return multierr.Combine(logPushErrors...)
+	}
 	return nil
 }
+
+func (l *LogsExporter) logToEntry(
+	log pdata.LogRecord,
+	attributes map[string]interface{},
+	mr *monitoredres.MonitoredResource,
+) (logging.Entry, error) {
+	entry := logging.Entry{
+		Timestamp: log.Timestamp().AsTime(),
+		Severity:  logging.Severity(log.SeverityNumber()),
+		Trace:     log.TraceID().HexString(),
```

Comment:
I think Cloud Logging `LogEntry.trace` should be prefixed by `projects/{projectId}/traces` (https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry); otherwise, `trace` value will be different between GCP infrastructure log (such as AppEngine request log, etc.) and application log.

Reply:
You're probably right. If it's okay with you @damemi I'll probably punt Trace off of this PR as well since you and your team are probably better equipped to do this part right.

Reply:
@braydonk sounds good, we're trying to iterate on this rather than get it perfect all at once :)

Reply:
I'll mark this conversation resolved, we'll keep a note of it as something to look into next.